scripts/simplify_network.py

```diff
@@ -390,22 +386,29 @@ def consense(x):
         snakemake = mock_snakemake('simplify_network', simpl='', network='elec')
     configure_logging(snakemake)
 
-    n = pypsa.Network(snakemake.input.network)
+    paths, config, wildcards, logs, out = retrieve_snakemake_keys(snakemake)
+
+    n = pypsa.Network(paths.network)
 
     n, trafo_map = simplify_network_to_380(n)
 
-    n, simplify_links_map = simplify_links(n)
+    Nyears = n.snapshot_weightings.objective.sum() / 8760
+    technology_costs = load_costs(tech_costs = paths.tech_costs,
+                                  config = config['costs'],
+                                  elec_config = config['electricity'], Nyears = Nyears)
```

Comment:
I think here you missed to remove the keyword arguments and add them as positional arguments?